### PR TITLE
Fixed typo in setup.py, fixed bug where predict fails if output direc…

### PR DESCRIPTION
**Pull request by EuanPyle**
…tory already exists, added functionality to allow user to provide a list of tomograms to predict in predict_config.json

### CODE REVIEW

**Comment by EuanPyle** [comment]
PR which fixes a couple of errors: 
added a missing comma in setup.py, 
added a line to allow users to output tomograms in a directory that already exists without error
added a few lines to allow users to provide a list of tomograms in predict_config.json rather than either specifying the directory of even/odd tomograms or providing a string for the path to one specific tomogram

**Comment by thorstenwagner** [comment]
Sorry, I didnt noticed this pull request.

Could you change

```
if not os.path.exists(config['output']):
                os.makedirs(config['output'])
```
to
```
os.makedirs(config['output'], exists_ok=True)
```

File: cryocare/scripts/cryoCARE_predict.py [hunk 1]
```diff
@@ -91,11 +91,14 @@ def main():
                 norm_data = json.load(f)
                 mean = norm_data["mean"]
                 std = norm_data["std"]
-            os.makedirs(config['output'])
+            if not os.path.exists(config['output']):
```
Comment (thorstenwagner) [inline cryocare/scripts/cryoCARE_predict.py:94]: Better use
```
os.makedirs(config['output'], exist_ok=True)
```

**Review by thorstenwagner — CHANGES_REQUESTED** [review]
Sorry, I didnt noticed this pull request. Thank you very much!!

Just make this minor change and we can merge it :-)

File: cryocare/scripts/cryoCARE_predict.py (hunk 1 above)
Comment (thorstenwagner) [reply cryocare/scripts/cryoCARE_predict.py:94]: Well, I will simply change it :-)

**Pull request merged**